src/backend/backend.cpp

```diff
@@ -19,6 +19,8 @@ namespace big
 {
 	void backend::loop()
 	{
+		for (auto& command : g_bool_commands)
+			command->refresh();
 		for (auto& command : g_looped_commands)
 			command->refresh();
```

Comment:
double refresh?

Reply:
One for bool commands and one for looped commands.